src/test/java/org/prgrms/kdt/customer/JdbcCustomerManagerTest.java

```diff
@@ -0,0 +1,109 @@
+package org.prgrms.kdt.customer;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.prgrms.kdt.JdbcBase;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class JdbcCustomerManagerTest extends JdbcBase {
+
+    @Autowired
+    private CustomerManager customerManager;
+
+    @BeforeEach
+    void init() {
+        customerManager.deleteAll();
+    }
+
+    @DisplayName("고객은 저장될 수 있다.")
+    @Test
+    void saveTest() {
+        // given
+        String customerName = "test1";
+        Customer customer = new Customer(customerName);
+
+        // when
+        Customer savedCustomer = customerManager.save(customer);
+
+        // then
+        assertThat(customerManager.findById(savedCustomer.getId()).isPresent())
+                .isTrue();
```

Comment:
then이 너무 어렵습니다..!
```suggestion
        var customer = customerManager.findById(savedCustomer.getId())
                .get()
        assertThat(customer).isNotNull()
```

이런형태가 더 읽기좋지않을까요?

Reply:
optional 안의 값이 null 일 수도 있다고 생각해서 조금 복잡하게 된 것 같습니다. 테스트 코드에서는 가독성을 위해서 optional 값을 바로 빼는 것이 좋을까요?

Reply:
확정적으로 null이 아니라면, 꺼내서(`get()`) 검증하려는거에 집중하고 있습니다. (개인적인 선호도입니다 ㅎㅎ)